indico/modules/auth/templates/emails/create_local_identity.txt

```diff
@@ -0,0 +1,18 @@
+{% extends 'emails/base_i18n.txt' %}
+
+{% block subject -%}
+    {% trans %}Create local account{% endtrans %}
+{%- endblock %}
+
+{% block header_recipient -%}
+    {{ user.first_name }}
+{%- endblock %}
+
+{% block body -%}
+{% trans %}We heard that you forgot your Indico username or password.{% endtrans %}
+
+{% trans %}We found an Indico account, but it has no username/password set.{% endtrans %}
```

Comment:
Maybe we could mention the name of the provider (assuming we have a nice display name)? People might click that forgotten password link when they forget that they login with some external provider and this could remind them.

Reply:
good idea, was considering it anyway ;)